docker/dockerfile

```diff
@@ -1,15 +1,3 @@
-#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why are you removing the Apache 2 license notice?

Reply:
I must have removed it accidentally... sorry